src/test/resources/schema/transportDocument.json

```diff
@@ -1,1022 +1,610 @@
-                      "streetNumber": {
-                        "maxLength": 50,
-                        "type": "string",
-                        "description": "The number of the street of the partyÔÇÖs address."
-                      },
-                      "floor": {
-                        "maxLength": 50,
-                        "type": "string",
-                        "description": "The floor of the partyÔÇÖs street number."
-                      },
-                      "postCode": {
-                        "maxLength": 10,
-                        "type": "string",
-                        "description": "The post code of the partyÔÇÖs address."
-                      },
-                      "city": {
-                        "maxLength": 65,
-                        "type": "string",
-                        "description": "The city name of the partyÔÇÖs address."
-                      },
-                      "stateRegion": {
-                        "maxLength": 65,
-                        "type": "string",
-                        "description": "The state/region of the partyÔÇÖs address."
-                      },
-                      "country": {
-                        "maxLength": 75,
-                        "type": "string",
-                        "description": "The country of the partyÔÇÖs address."
-                      }
-                    }
-                  }
+  "required": [
+    "shippingInstruction",
+    "transportDocumentReference"
+  ],
+  "type": "object",
+  "description": "The document that governs the terms of carriage between shipper and carrier for maritime transportation. Two distinct types of transport documents exist:\n- Bill of Lading\n- Sea Waybill. \n",
+  "properties": {
+    "transportDocumentReference": {
+      "maxLength": 20,
+      "type": "string",
+      "description": "A unique number allocated by the shipping line to the transport document and the main number used for the tracking of the status of the shipment.\n"
+    },
+    "transportDocumentCreatedDateTime": {
+      "type": "string",
+      "description": "Date and time when the TransportDocument was created\n",
+      "format": "date-time"
+    },
+    "transportDocumentUpdatedDateTime": {
+      "type": "string",
+      "description": "Date and time when the TransportDocument was updated\n",
+      "format": "date-time"
+    },
+    "issueDate": {
+      "type": "string",
+      "description": "Date when the transport document has been issued"
+    },
+    "shippedOnBoardDate": {
+      "type": "string",
+      "description": "Date when the last container that is linked to the transport document is physically loaded onboard the vessel indicated on the transport document."
+    },
+    "receivedForShipmentDate": {
+      "type": "string",
+      "description": "Date when the last container linked to the transport document is physically in the terminal (customers cleared against the intended vessel)."
```

Comment:
How come we no longer schema validate these as being a date?

Reply:
Because the tests kept failing because it claimed it didn't support those formats, but now it seems to work for some reason. Reverting that change.

Reply:
Should be fixed now